docs/faq/admin.txt

```diff
@@ -97,11 +97,15 @@ like to make should be possible by editing the stylesheet.
 What browsers are supported for using the admin?
 ================================================
 
-The admin provides a fully-functional experience to `YUI's A-grade`_ browsers,
-with the notable exception of IE6, which is not supported.
+The admin provides a fully-functional experience to the recent versions of
+modern, web standards compliant browsers. On desktop this means Edge, Firefox,
+Chrome, Safari, and others.
```

Comment:
I would alphabetize browsers' names and add _Opera_ (I know we have _"others"_, but this is the only major browser that we omitted).